.github/CODEOWNERS

```diff
@@ -630,6 +630,7 @@
 
 
 # Frontend unit tests
+/core/templates/components/entity-creation-services/story-creation.service.spec.ts @nithusha21 @U8NWXD
```

Comment:
Can we use `/core/templates/components/**/*.spec.ts` here?
/cc @nithusha21

Reply:
i dont understand you

Reply:
you mean i should remove the ` text entity-creation-servicess/story-creation.service`